lib/credo/check/readability/pipes_to_anonymous_functions.ex

```diff
@@ -0,0 +1,55 @@
+defmodule Credo.Check.Readability.PipesToAnonymousFunctions do
+  @moduledoc false
+
+  @checkdoc """
+  Do not pipe into anonymous functions.
+
+  So while this is fine:
+      foo
+      |> times_2()
+      |> times_2()
+  The code in this example ...
+      foo
+      |> (fn i -> i * 2 end).()
+      |> (fn i -> i * 2 end).()
+  ... should be refactored to look like this:
+      defp timex_2(i), do: i * 2
+
+      foo
+      |> times_2()
+      |> times_2()
+  Piping into anonymous functions is cluttered and difficult to read.
+  Instead, define a private function and use that instead.
```

Comment:
Repetition of "instead" here. :)

```suggestion
  Instead, define a private function and use that.
```